src/gc/handletablecore.cpp

```diff
@@ -516,14 +516,14 @@ BOOL SegmentInitialize(TableSegment *pSegment, HandleTable *pTable)
 
 #ifndef FEATURE_REDHAWK // todo: implement SafeInt
     // Prefast overflow sanity check the addition
-    if (!ClrSafeInt<uint32_t>::addition(dwCommit, g_SystemInfo.dwPageSize, dwCommit))
+    if (!ClrSafeInt<uint32_t>::addition(dwCommit, OS_PAGE_SIZE, dwCommit))
```

Comment:
Ah, I have not realized that the change to support larger page sizes on Unix is not in yet and that that change will solve that (it defines OS_PAGE_SIZE as a call to a new GCToOSInterface::GetPageSize()). So please ignore my comment.